src/Talis/Persona/Client/Base.php

```diff
@@ -289,7 +260,8 @@ protected function createRequest($url, array $opts)
             $httpConfig['headers']['Content-Type'] = 'application/x-www-form-urlencoded';
         }
 
-        $client = $this->getHTTPClient($opts['skipRevalidation']);
+        // TODO: turn off all caching
```

Comment:
do the plugins need configuring? I believe the cache plugins are defaulted, but the no-cache header is set. Review the behavior.